src/main/java/Duke.java

```diff
@@ -1,10 +1,44 @@
+import command.Parser;
```

Comment:
I like how you explicitly list the individual classes to be imported -- it makes it easier to see which exact classes are being used in this file :)